lib/PuppeteerSharp.sln

```diff
@@ -1,22 +1,23 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15

```

Comment:
let me know if u want the sln reverted back to VS15

Reply:
This is perfect!